utils/python/CIME/build.py

```diff
@@ -411,36 +411,35 @@ def build_checks(case, build_threaded, comp_interface, use_esmf_lib, debug, comp
     case.set_value("BUILD_COMPLETE", False)
 
     case.flush()
+    return sharedpath
```

Comment:
I've added a comment, it should be pointed out that this is an improvement over the prior behavior which was to set and read in an environment variable.